src/CodeReport/AssemblyTestLiterals.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <string_view>
+
+namespace orbit_code_report_internal {
```

Comment:
Why the `_internal` here? This is a private header.

Reply:
I don't really care. I will change it. :-)

Reply:
Done.